yorkie-intelligence/.pre-commit-config.yaml

```diff
@@ -0,0 +1,9 @@
+repos:
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  # Ruff version.
+  rev: v0.9.2
+  hooks:
+    # Run the linter.
+    - id: ruff
+    # Run the formatter.
+    - id: ruff-format
```

Comment:
Need [POSIX newline](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) here.